packages/wdio-spec-reporter/src/index.ts

```diff
@@ -366,7 +374,7 @@ export default class SpecReporter extends WDIOReporter {
                 const testIndent = `${DEFAULT_INDENT}${suiteIndent}`
 
                 // Output for a single test
-                output.push(`${testIndent}${chalk[this.getColor(state)](this.getSymbol(state))} ${testTitle.trim()}`)
+                output.push(`${testIndent}${this.setMessageColor(this.getSymbol(state), state!)} ${testTitle.trim()}`)
```

Comment:
instead of ignoring the type issue, how about we define the state variable like this:
```ts
const state = test.state as State
```

Reply:
resolved